src/pretix/control/views/organizer.py

```diff
@@ -2016,6 +2016,8 @@ class LogView(OrganizerPermissionRequiredMixin, PaginationMixin, ListView):
     context_object_name = 'logs'
 
     def get_queryset(self):
+        # technically, we'd also need to sort by pk since this is a paginated list, but in this case we just can't
+        # bear the performance cost
```

Comment:
What about replacing the datetime-index with (datetime, id)-index? Would that improve performance? AFAIK a pure datetime-based access would/could still use the (datetime, id)-index.